### PR TITLE
translate example list

### PR DESCRIPTION
**Pull request by sunny0826**
translate example list.
Is [breakfast](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/breakfast.md) serious?

### CODE REVIEW

File: examples/README-CN.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# 示例
```
Comment (Liujingfang1) [inline examples/README-CN.md]: Similar to the docs, let's create a `zh` sub directory under `examples` and put all the Chinese examples inside `zh`.

File: examples/README-CN.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+这些示例默认 `kustomize` 在您的 `$PATH` 中。
+
+这些示例通过了 [pre-commit](../bin/pre-commit.sh) 测试，并且应该与 HEAD 一起使用。
+
+<!-- @installkustomize @test -->
+```
+go get sigs.k8s.io/kustomize
+```
+
+ * [hello world](helloWorld/README.md) - 部署多个配置不同的 Hello World 服务。
```
Comment (Liujingfang1) [inline examples/README-CN.md]: 部署多个不同配置的 Hello World 服务

File: examples/README-CN.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+ * [last mile helm](chart.md) - 对 helm chart 进行 last mile 修改。
+   
+ * [LDAP](ldap/README.md) - 部署多个配置不同的 LDAP 服务。
```
Comment (Liujingfang1) [inline examples/README-CN.md]: 配置不同 --》 不同配置

File: examples/README-CN.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+ * [mySql](mySql/README.md) - 从头开始创建一个 MySQL 的生产配置。
+
+ * [springboot](springboot/README.md) - 从头开始创建一个 Spring Boot 项目的生产配置。
+
+ * [combineConfigs](combineConfigs.md) -
+   混合来自不同用户的配置数据（例如来自 devops/SRE 和 developers）。
```
Comment (Liujingfang1) [inline examples/README-CN.md]: 混合 -》 融合

File: examples/README-CN.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+   
+ * [configGenerations](configGeneration.md) - 当 ConfigMapGenerator 修改时进行滚动更新。
+
+ * [secret generation](kvSourceGoPlugin.md) - 生成 Secret。
+ 
+ * [generatorOptions](generatorOptions.md) -修改所有 ConfigMapGenerator 和 SecretGenerator 的行为。
+
+ * [breakfast](breakfast.md) - 给 Alice 和 Bob 定制一顿早餐 :)
```
Comment (Liujingfang1) [inline examples/README-CN.md]: Yes, this example is serious :)
It demonstrates that Kustomize works with any types as long as those YAML follows kubernetes styles, e.g. containing apiVersion, kind, metadata.name.

File: examples/README-CN.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+   
+ * [vars](wordpress/README.md) - 通过 vars 将 k8s 运行时数据注入容器参数（例如，为 wordpress 指定 SQL 服务）。
```
Comment (Liujingfang1) [inline examples/README-CN.md]: 建议改成：
通过 vars 将一个资源的数据注入另一个资源的容器参数 （例如，为 wordpress 指定 SQL 服务）。

File: examples/README-CN.md (hunk 5 above)
Comment (monopole) [reply examples/README-CN.md]: If breakfast isn't appropriate, we could refactor it into lunch.

Comment (sunny0826) [reply examples/README-CN.md]: > If breakfast isn't appropriate, we could refactor it into lunch.

Breakfast is great. I love breakfast..._(´ཀ`」 ∠)_...Congratulations Coffee and  become k8s resource.

File: examples/zh/README-CN.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+这些示例默认 `kustomize` 在您的 `$PATH` 中。
+
+这些示例通过了 [pre-commit](../bin/pre-commit.sh) 测试，并且应该与 HEAD 一起使用。
+
+<!-- @installkustomize @test -->
+```
+go get sigs.k8s.io/kustomize
+```
+
+ * [hello world](helloWorld/README.md) - 部署多个不同配置的 Hello World 服务。
```
Comment (Liujingfang1) [inline examples/zh/README-CN.md]: helloWorld/README.md --> ../helloWorld/README.md

Same for the following links.

Comment (sunny0826) [reply examples/zh/README-CN.md]: My bad

**Pull request merged**